imgui-binding/src/main/native/jni_assertion.cpp

```diff
@@ -0,0 +1,48 @@
+#include "jni_assertion.h"
+
+static JavaVM* assertJvm;
+
+jmethodID jImAssertCallbackMID = NULL;
+jobject jImAssertCallbackInstance = NULL;
+
+namespace Jni
+{    
+    void InitAssertion(JNIEnv* env) {
+        env->GetJavaVM(&assertJvm);
+        
+        jclass jImAssertCallback = env->FindClass("imgui/assertion/ImAssertCallback");
+        jImAssertCallbackMID = env->GetMethodID(jImAssertCallback, "imAssert", "(Ljava/lang/String;ILjava/lang/String;)V");
+    }
+    
+    JNIEnv* GetAssertEnv() {
```

Comment:
`Jni` namespace already has a method to get a `JNIEnv*` class. It will be cleaner to include the common header and reuse already existing code. Same can be done for the `JavaVM` field.

Reply:
I initially tried that, the problem is that the imconfig.h header gets included before the rest of the imgui defines, which causes a compilation error in the _common.cpp when trying to lookup the ImVec* classes.

Reply:
An idea to fix it, but it would require some rework of the _common.cpp, is to move the ImVec stuff to another header that does not get included in _binding.h. But that approach gets very messy fast. Another option is to pull out the `GetEnv()` function into the `jni_assertions.h`, or into its own header.

Reply:
I've considered about moving JVM specific stuff into a separate header too. Seems legit.

Reply:
Sounds good, I'll look at implementing that.

Reply:
How does that look?